docs/usage/working-with-bundlers.md

```diff
@@ -61,26 +59,50 @@ hello_python().then((result) => {
 This should be sufficient for Vite dev mode:
 
 ```
-$ npx vite
+npx vite
 ```
 
-For a production build, you must also manually make sure that all Pyodide files
-will be available in `dist/assets`, by first copying them to `public/assets`
-before building:
+For a production build, you must also make sure that all Pyodide files
+will be available in `dist/assets` by using a Vite plugin like this in your
+`vite.config.js` file:
 
-```
-$ mkdir -p public/assets/
-$ cp node_modules/pyodide/* public/assets/
-$ npx vite build
+```js
+import { defineConfig } from "vite";
+import { copyFile, mkdir } from "fs/promises";
+import { join } from "path";
+
+export default defineConfig({
+  optimizeDeps: { exclude: ["pyodide"] },
+  plugins: [
```

Comment:
Given that you're adding to `plugins` anyway, can you remove the above shorter `vite.config.js` file since it's made redundant with this, and probably just move the prose about `optimizeDeps` down here? That way someone doesn't accidentally copy just the shorter `vite.config.js` file from above, which doesn't quite work.